src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -148,6 +166,74 @@ private <T> List<T> loadData(final String configKey, Type type)
 		}
 	}
 
+	private List<InventorySetup> loadV1Setups() {
+		Type setupTypeV1 = new TypeToken<ArrayList<InventorySetup>>()
+		{
+
+		}.getType();
+		return loadData(CONFIG_KEY_SETUPS, setupTypeV1);
+	}
+
+	private List<InventorySetup> loadV2Setups() {

```

Comment:
Check style gives me a warning about curly braces on same line. Could you move these to below the function?

Reply:
done